microsoft/testsuites/vm_extensions/linux_patch_extension.py

```diff
@@ -14,17 +14,57 @@
     simple_requirement,
 )
 from lisa.base_tools.service import Service
-from lisa.operating_system import BSD
+from lisa.operating_system import BSD, SLES, CBLMariner, Debian, Redhat
 from lisa.sut_orchestrator import AZURE
 from lisa.sut_orchestrator.azure.common import (
     get_compute_client,
     get_node_context,
     wait_operation,
 )
 from lisa.sut_orchestrator.azure.platform_ import AzurePlatform
+from lisa.util import SkippedException, UnsupportedDistroException
+
+
+def _is_os_unsupported(self, node: Node) -> None:
+    """
+    Below specific Linux distro versions that are
+    not supported by linux patch extension.
+    debian debian-11 11 0.20230124.1270
+    redhat rhel-arm64 8_6-arm64 8.6.2022060901
+    debian debian-11 11-backports 0.20221219.1234
+    microsoftcblmariner cbl-mariner cbl-mariner-2-arm64 2.20230126.01
+    suse sles-15-sp5 gen2 2024.02.07
+    """
+    unsupported_versions_x86_64: Dict[type, List[str]] = {
+        Debian: [10, 11],
+        SLES: [15],
+        CBLMariner: [2],
+    }
+    unsupported_versions_arm64: Dict[type, List[str]] = {
+        Redhat: [8_6],
+    }
+
+    arch = node.os.get_kernel_information().hardware_platform  # type: ignore
+    if arch == "aarch64":
+        unsupported_versions = unsupported_versions_arm64
+    else:
+        unsupported_versions = unsupported_versions_x86_64
+
+    # Extract Debian version from the complete version string
+    debian_version = node.os.information.version.major.split()[1]
+
+    for distro, version_list in unsupported_versions.items():
+        if isinstance(node.os, distro) and debian_version in version_list:
```

Comment:
`if isinstance(node.os, distro) and node.os.information.version.major in version_list:`

Reply:
logic changed